CONTRIBUTING.md

```diff
@@ -1,96 +1,60 @@
-  - [Frontend style guide](https://github.com/grafana/grafana/tree/master/style_guides)
+For more ways you can contribute, check out the [Open Source Guides](https://opensource.guide/how-to-contribute/).
 
-## Steps to contribute
+### Report bugs
 
-Should you wish to work on a GitHub issue, check first if it is not already assigned to someone. If it is free, you claim it by commenting on the issue that you want to work on it. This is to prevent duplicated efforts from contributors on the same issue.
+Report a bug by submitting a [bug report](https://github.com/grafana/grafana/issues/new?labels=type%3A+bug&template=1-bug_report.md). Make sure that you provide as much information as possible on how to reproduce the bug.
 
-Please check the [`beginner friendly`](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22) and [`help wanted`](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22) labels to find issues that are good for getting started. If you have questions about one of the issues, with or without the tag, please comment on them and one of the core team or the original poster will clarify it.
+Before submitting a new issue, try to make sure someone hasn't already reported the problem. Look through the [existing issues](https://github.com/grafana/grafana/issues) for similar issues.
 
-To set up a local development environment we recommend reading [Building Grafana from source](http://docs.grafana.org/project/building_from_source/).
+#### Security issues
 
-## Pull request checklist
+If you believe you've found a security vulnerability, please read our [security policy](https://github.com/grafana/grafana/security/policy) for more details.
 
-Whether you are contributing or doing code review, first read and understand https://google.github.io/eng-practices/review/reviewer/ for general engineering practices around code reviews that we also use.
+### Suggest enhancements
 
-- Branch from the master branch and, if needed, rebase to the current master branch before submitting your pull request. If it doesn't merge cleanly with master, then you might be asked to rebase your changes.
+If you have an idea of how to improve Grafana, submit an [enhancement request](https://github.com/grafana/grafana/issues/new?labels=type%3A+feature+request&template=2-feature_request.md).
 
-- If your patch is not getting reviewed or you need a specific person to review it, then you can @-reply a reviewer asking for a review in the pull request or a comment.
+### Your first contribution
 
-- Add tests relevant to the fixed bug or new feature.
+Unsure where to begin contributing to Grafana? Start by browsing issues labeled `beginner friendly` or `help wanted`.
 
-### High-level checks
+- [Beginner-friendly](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22) issues shouldn't require more than changing a small number of lines.
```

Comment:
```suggestion
- [Beginner-friendly](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22) issues are generally are straightforward to complete.
```

What do you think about this phrasing? I don't think I keep in mind the number of lines to be changed as we label this issues but instead the complexity of it.

Reply:
I tried to be a bit more specific to avoid words like "simple" and "easy". I can't decide whether what's seen as "straightforward" by one, might not be for another.

Reply:
I recognise this is a delicate topic. That's why I added the word _generally_ - I think this covers for those cases where one might disagree.

Reply:
In my eyes, it reads "we think these are relatively straightforward but there's room for disagreement"

Reply:
```suggestion
- [Beginner-friendly](https://github.com/grafana/grafana/issues?q=is%3Aopen+is%3Aissue+label%3A%22beginner+friendly%22) issues are generally straightforward to complete.
```

Reply:
I'm fine with that, just removed an extra "are"